src/BlockInfo.h

```diff
@@ -70,6 +70,9 @@ class cBlockInfo
 	/** Sound when placing this block */
 	AString m_PlaceSound;
 
+	/** Defines how long it takes to break the block */
```

Comment:
That needs further investigation. They are the numbers from net.minecraft.block.Block.blockHardness.